src/typehints/transformers/models.pyi

```diff
@@ -12,3 +12,34 @@ class AutoTokenizer:
     @classmethod
     def from_pretrained(cls, model_type: str) -> AutoTokenizer: ...
     def __call__(self, text: str, return_tensors: str) -> torch.Tensor: ...
+
+class PretrainedConfig:
+    def __init__(self, **kwargs): ...
+
+class EncodecConfig(PretrainedConfig):
+    sampling_rate: int
+
+class MusicgenDecoderConfig(PretrainedConfig):
+    pass
```

Comment:
I don't like a little doing this `pass` here, would it work with just `...` instead? 
```suggestion
class MusicgenDecoderConfig(PretrainedConfig):
    ...
```

Reply:
Good point, done!